ldk/csharp/OliveHelpsLDK/SelfTestLoop/TestBuilder.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text.Json;
+using OliveHelpsLDK;
+using OliveHelpsLDK.Logging;
+using SelfTestLoop.Tests;
+
+namespace SelfTestLoop
+{
+    public class TestBuilder
+    {
+        public ILogger Logger { get; set; }
```

Comment:
Does Logger has to be public here or have setter for that matter? Generally you don't want to allow changing state of injected members. Perhaps I'm missing the purpose.